source/docs/magento2/headless-payments.md

```diff
@@ -157,4 +160,31 @@ Below is an example from Front-Commerce's core:
 
 <blockquote class="note">
 We encourage you to investigate existing Adapters' source code from [Front-Commerce's core](https://gitlab.com/front-commerce/magento2-module-front-commerce/-/tree/master/app/code/FrontCommerce/Integration/Model/HeadlessPayments/Adapter) to learn about advanced patterns.
-</blockquote>
\ No newline at end of file
+</blockquote>
+
+### Allow the Payment's URLs
```

Comment:
```suggestion
## Allow the Payment's URLs
```

I think it should be at the same level than "Register the payment adapter". See https://deploy-preview-150--heuristic-almeida-1a1f35.netlify.app/docs/magento2/headless-payments.html#Allow-the-Payment%E2%80%99s-URLs